.github/workflows/nightly.yml

```diff
@@ -94,3 +94,6 @@ jobs:
       s3_bucket: ${{ secrets.IT_CASE_S3_BUCKET }}
       s3_access_key: ${{ secrets.IT_CASE_S3_ACCESS_KEY }}
       s3_secret_key: ${{ secrets.IT_CASE_S3_SECRET_KEY }}
+  python-wheels:
```

Comment:
I'm wondering whether we should just inline the two jobs. Adding them as a template makes the workflow appear on the left side of the workflow list in GitHub Actions (analogously to [the entry for the template.flink-ci.yml](https://github.com/apache/flink/actions/workflows/template.flink-ci.yml) without any additional utilization.

One argument to keep the template workflow could be that we would be able to enable `workflow_dispatch` for the wheel generation. But I'm not aware of such a use-case. :thinking: 

WDYT? :thinking: